app/serializers/contributor.ts

```diff
@@ -1,23 +1,10 @@
 import DS from 'ember-data';
-
 import OsfSerializer from './osf-serializer';
 
 export default class ContributorSerializer extends OsfSerializer {
-    serialize(snapshot: DS.Snapshot, options = {}) {
-        // Restore relationships to serialized data
+    serialize(snapshot: DS.Snapshot, options: {}) {
```

Comment:
The `seralize` hook can be removed when https://openscience.atlassian.net/browse/ENG-2596 is done.